docs/getting-started.md

```diff
@@ -72,8 +72,7 @@ Alternatively, secrets can be mounted directly into the Relay Proxy. Secrets are
 # values.yaml
 relay:
    secrets:
-      - volumeName: tls-cert
```

Comment:
volumeName is no longer used. Setting it won't cause a problem. We will just ignore it.